pyp2spec/pyp2conf.py

```diff
@@ -220,6 +220,27 @@ def archive_name(self, version):
         archive_name = edited_sdist_filename.replace("-" + version, "")
         return archive_name
 
+    def extras(self):
+        """Return the list of the found extras names.
+
+        Packages define extras explicitly via `Provides-Extra` and
+        indirectly via `Requires-Dist` metadata.
+        PyPI metadata doesn't provide the first one, but it is possible to
+        derive extras names from the `requires_dist` key.
+        Example value: ["sphinxcontrib-websupport ; extra == \"docs\"",
+        "flake8>=3.5.0 ; extra == \"lint\""]
+        """
+
+        extras = set()
+        requires_dist = self.package_data["info"]["requires_dist"]
```

Comment:
This also suffers from https://github.com/befeleme/pyp2spec/issues/41

Reply:
Fixed in https://github.com/befeleme/pyp2spec/pull/42